src/constants.ts

```diff
@@ -43,3 +53,36 @@ export const allSaasOfferings = saasOfferings;
 
 _.set(window, 'OPENSHIFT_CONSTANTS.SERVICE_CATALOG_CATEGORIES', categories);
 _.set(window, 'OPENSHIFT_CONSTANTS.SAAS_OFFERINGS', saasOfferings);
+
+
+var resources: any = {
+  description: 'Lorem ipsum dolor sit amet, consectular adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud',
+  links: [
+    {
+      title: 'Welcome',
+      help: 'default'
+    },
+    {
+      title: 'Building a new Application',
+      help: 'new_app'
+    },
+    {
+      title: 'Authorization',
+      help: 'authorization'
+    },
+    {
+      title: 'Deployments',
+      help: 'deployments'
+    },
+    {
+      title: 'Application Health',
+      help: 'application_health'
+    },
+    {
+      title: 'Visit us on Facebook',
+      href: 'http://facebook.com/RedHatInc'
+    }
+  ]
+};
+
+_.set(window, 'OPENSHIFT_CONSTANTS.RESOURCES', resources);
```

Comment:
Should have commented on first review, but we should really give a name that isn't so generic. Maybe `CATALOG_HELP_RESOURCES` or similar.